.circleci/config.yml

```diff
@@ -428,20 +428,63 @@ jobs:
       TERM: xterm
       CMAKE_BUILD_TYPE: Debug
       CMAKE_OPTIONS: -DLLL=ON
+      CMAKE_PROGRAM_PATH: /Users/distiller/project/z3/bin
+      CMAKE_LIBRARY_PATH: /Users/distiller/project/z3/bin
+      CMAKE_INCLUDE_PATH: /Users/distiller/project/z3/include
     steps:
       - checkout
+      - restore_cache:
+          keys:
+            - dependencies-osx-{{ .Branch }}-{{ checksum ".circleci/config.yml" }}
       - run:
           name: Install build dependencies
           command: |
             brew unlink python
-            brew install z3
             brew install boost
             brew install cmake
             brew install wget
             ./scripts/install_obsolete_jsoncpp_1_7_4.sh
+            brew install coreutils
+            wget https://github.com/Z3Prover/z3/releases/download/Z3-4.8.5/z3-4.8.5-x64-osx-10.14.2.zip
+            unzip z3-4.8.5-x64-osx-10.14.2.zip
+            rm -f z3-4.8.5-x64-osx-10.14.2.zip
+            mv z3-4.8.5-x64-osx-10.14.2 z3
+            rm z3/bin/*.dylib
+            wget https://github.com/ethereum/evmone/releases/download/v0.1.0/evmone-0.1.0-darwin-x86_64.tar.gz
+            tar xzpf evmone-0.1.0-darwin-x86_64.tar.gz
+            rm -f evmone-0.1.0-darwin-x86_64.tar.gz
+      - save_cache:
```

Comment:
Shouldn't we rather have a docker image?

Reply:
would be nice, but as far as I know docker is tightly bounded the the linux kernel.. i think it is not possible to run a macos inside a docker

Reply:
*slapsforehead* I cannot get away with saying that it was before my first coffee, can I? :)

Reply:
no prob :)